Makefile

```diff
@@ -173,6 +182,25 @@ create-aks-cluster-for-karpenter: ## Create test AKS cluster (with msi, cilium,
     --enable-managed-identity --enable-oidc-issuer --enable-workload-identity -o none
 	az aks get-credentials --name $(AZURE_CLUSTER_NAME) --resource-group $(AZURE_RESOURCE_GROUP) --overwrite-existing
 
+## --------------------------------------
+## AWS resources
+## --------------------------------------
+
+.PHONY: deploy-aws-cloudformation
+deploy-aws-cloudformation: ## Deploy AWS CloudFormation stack
+	curl -fsSL https://raw.githubusercontent.com/aws/karpenter-provider-aws/v"${AWS_KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml  > "${TEMPOUT}" \
+	&& aws cloudformation deploy \
+	--stack-name "Karpenter-${CLUSTER_NAME}" \
+	--template-file "${TEMPOUT}" \
+	--capabilities CAPABILITY_NAMED_IAM \
+	--parameter-overrides "ClusterName=${CLUSTER_NAME}"
+
+.PHONY: create-eks-cluster
+create-eks-cluster: ## Create test EKS cluster
+	@envsubst < $(CLUSTER_CONFIG_FILE) > $(CLUSTER_CONFIG_FILE).tmp && mv $(CLUSTER_CONFIG_FILE).tmp $(CLUSTER_CONFIG_FILE)
```

Comment:
please don't overwrite this file in repo. I suggest you rename `./docs/aws/clusterconfig.yaml` to `./docs/aws/clusterconfig.yaml.template`.
When you make this target, render it to a file  `./docs/aws/clusterconfig.yaml` from this template.